docs/how-tos/build-from-git-repo.rst

```diff
@@ -14,6 +14,15 @@ you can do the following:
 This will clone the ``stevedore`` repository and build the package from the
 local copy.
 
+You can also use the ``bootstrap-parallel`` command for faster builds:
+
+.. code-block:: console
+
+   $ fromager bootstrap-parallel stevedore @ git+https://github.com/openstack/stevedore.git
```

Comment:
I think this syntax isn't right because there are spaces. It would at least need to be quoted to be processed properly, as was done in the test script. Please submit a follow up PR to fix that.

```suggestion
   $ fromager "bootstrap-parallel stevedore @ git+https://github.com/openstack/stevedore.git"
```